src/interfaces/issues-processor-options.ts

```diff
@@ -51,4 +51,7 @@ export interface IIssuesProcessorOptions {
   ignoreIssueUpdates: boolean | undefined;
   ignorePrUpdates: boolean | undefined;
   exemptDraftPr: boolean;
+  mlflow: boolean;
```

Comment:
This option is only set to true in `src/main.ts` and `__tests__/mlflow.spec.ts` to avoid breaking the existing tests.